src/main/resources/org/springframework/integration/aws/config/xml/spring-integration-aws-1.0.xsd

```diff
@@ -61,165 +75,175 @@
 		</xsd:attribute>
 	</xsd:attributeGroup>
 
-	<xsd:element name="s3-outbound-channel-adapter">
-		<xsd:complexType>
+	<xsd:complexType name="outboundS3BaseType">
+		<xsd:choice minOccurs="0" maxOccurs="2">
+			<xsd:element ref="integration:poller" minOccurs="0" maxOccurs="1"/>
+			<xsd:element name="request-handler-advice-chain" type="integration:handlerAdviceChainType" minOccurs="0" maxOccurs="1" />
```

Comment:
If you mean the fresh PR for the `<request-handler-advice-chain>` according to the https://jira.spring.io/browse/INTEXT-187 requirements and just only for the 1.0.0 master, then that sounds great and I'd be happy to merge your first PR finally :wink: !